doc/content/integrations/pubsub/mqtt-client/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "MQTT Client"
+description: ""
+weight: 1
+---
+
+This section explains how to configure a Pub/Sub Integration using the built in MQTT client.
+
+<!--more-->
+
+Creating a Pub/Sub Integration with the built in MQTT client requires you to have an MQTT server running with an available endpoint. [Mosquitto](https://mosquitto.org/) is a popular open source MQTT Server you may use, but {{% tts %}} MQTT client works with any MQTT server.
+
+>No API key is needed for Pub/Sub messaging. All messages received by the Application Server are assumed to be authorized, so be sure to configure appropriate security on your MQTT server.
+
+In your application select the **Pub-subs** submenu from the **Integrations** side menu. Clicking on the **+ Add pub-sub** button will open the Pub/Sub creation screen.
```

Comment:
As mentioned in earlier review; it is exactly **PubSubs** and **PubSub**

Other styles; pub-sub, Pub/Sub, etc are wrong